### ISSUE TITLE
Unable to set autofocus attribute on <input> element when using useTranslation hook.

### ISSUE THREAD

**Issue by acecyq** [i18next/react-i18next#1088]
Am using this library together with i18next and i18next-xhr-backend. Am following the same i18n configurations as the sample code in the documentation.

**Comment by jamuhl** [comment]
what has the autoFocus todo with using a hook? Works on my app - did you set attribute "autoFocus"

feel free to provide a codesandbox for reproduction

**Comment by acecyq** [comment]
Sorry @jamuhl. Understand my problem statement was not clear and sounded illogical and I do believe that the hook will have nothing to do with a DOM event.

To answer your question I did set attribute autoFocus on the input and it did not work. Somehow when I stop importing useTranslation or Trans in the rendered components, autoFocus then started working.

When I tried to recreate the issue by creating a dummy app, and adding the i18next and react-i18next dependencies, the autoFocus attribute can be set on the input element when importing the useTranslation hook on the rendered component.

Will close this issue first as I feel it might be something else that is causing the issue.
